src/schema/schema.graphql

```diff
@@ -1610,7 +1612,7 @@ type ContactAttributes {
 	notes: String
 	thumbnailPhoto: String
 	image: ContactImage
-	userCertificate: String
+	userCertificate: StringOrArray
```

Comment:
graphql is a strict typed language, we shouldn't have a property whose type is unknown at build time

Reply:
why can't we always convert this to an array?

Reply:
ok